### ISSUE TITLE
Static array field count

### ISSUE THREAD

**Issue by roelvan** [Meteor-Community-Packages/meteor-autoform#462]
Thanks for this amazing package!

I have a question:

In my form the user can choose between 3 options that each have a specific number value. Depending on this value an arrayItem (for a [Object] type) should be added or removed dynamically.

Eg.: 
- Option 1: 1 participant - 120$
- Option 2: 2 participants - 200$
- ...

The user picks the second option, so now 2 participant form groups should appear. The user should not be able to alter this.

In a nutshell: I want to be able to control the adding and removing of array items myself and not let the user use the plus and minus button that is generated automatically.

I have been reading the docs all day, but wasn't able to solve this myself.

Would this be possible?

Thanks a bunch
